recipes/install_binary.rb

```diff
@@ -8,9 +8,11 @@
 
 include_recipe 'ark'
 
+node.default['packer']['zipfile'] = "packer_#{node['packer']['version']}_" \
+  "#{node['os']}_#{node['packer']['arch']}.zip"
```

Comment:
Perhaps a good compromise for both mine and Sanders thoughts would be to move the node.default into the default entry in the attributes file.